### ISSUE TITLE
New release tag

### ISSUE THREAD

**Issue by jwnimmer-tri** [meshcat-dev/meshcat#164]
It's been a long time since the more recent release tag.  We should probably make a new one.

(And probably, come up with a process for regular releases.)

**Comment by ManifoldFR** [comment]
Hi - sorry to hijack this issue - could the Python version also get a new release tag by the way? There's been an issue for this for a couple of months: meshcat-dev/meshcat-python#142

**Comment by jwnimmer-tri** [comment]
I think we can make a tag, but I'm not sure if anyone I know has access to publish on PyPI.  I'll ask around.

**Comment by ManifoldFR** [comment]
> I think we can make a tag, but I'm not sure if anyone I know has access to publish on PyPI. I'll ask around.

Shouldn't @rdeits have access ?

**Comment by rdeits** [comment]
Ah, sorry, happy to provide PyPI access so you can update that. I'll do
that next time I'm at my home computer.

**Comment by jwnimmer-tri** [comment]
That's great, thank you!  My PyPI username is the same: `jwnimmer-tri` (https://pypi.org/user/jwnimmer-tri/).

**Comment by rdeits** [comment]
Done!

**Comment by nhz2** [comment]
Once there is a new release, updating the artifact URL in MeshCat.jl to something more bit stable and without symlinks should fix the installation issue in https://github.com/rdeits/MeshCat.jl/pull/248

**Comment by ManifoldFR** [comment]
Hello, is there any progress towards a new release?

**Comment by jwnimmer-tri** [comment]
No, there isn't any progress.  To be notified when a release happens, you can go to https://github.com/meshcat-dev/meshcat and use the "Watch" widget option for "Custom" to select "Releases", and then GitHub will email you as soon as a release happens.
